data/kadi4mat_json/atmospheric_corrosion_models/din-corrosion-protection-model-iso-9223-compliant.json

```
@@ -0,0 +1,241 @@
+  "extras": [
+    {
+      "key": "Parameters",
+      "type": "dict",
+      "value": [
+        {
+          "key": "Time exponent",
+          "type": "list",
+          "value": [
+            {
+              "type": "str",
+              "value": "Use DIN recommended time exponents measured from the ISO CORRAG program"
+            },
+            {
+              "type": "str",
+              "value": "Enter manually"
+            }
+          ]
+        },
+        {
+          "key": "Corrosion type",
+          "type": "list",
+          "value": [
+            {
+              "type": "str",
+              "value": "Insignificant - Heated rooms with low relative humidity and insignificant air pollution, e.g., offices, schools, museums."
+            },
+            {
+              "type": "str",
+              "value": "Low - Unheated rooms with fluctuating temperature and relative humidity. Rare condensation and low air pollution, e.g., storage rooms, sports halls."
+            },
+            {
+              "type": "str",
+              "value": "Moderate - Rooms with occasional condensation and moderate air pollution from production processes, e.g., food processing plants, laundries, breweries, dairies."
+            },
+            {
+              "type": "str",
+              "value": "High - Rooms with frequent condensation and high air pollution from production processes, e.g., industrial plants, swimming pools."
+            },
+            {
+              "type": "str",
+              "value": "Very high - Rooms with very high frequency of condensation and/or high air pollution from production processes, e.g., mines, industrial cavities, non-ventilated halls in subtropical and tropical climate zones."
+            },
+            {
+              "type": "str",
+              "value": "Extreme - Rooms with nearly constant condensation or extended periods of exposure to extremely high humidity and/or high air pollution from production processes, e.g., non-ventilated halls in humid tropical climates with invasive pollution from outside air, including airborne chlorides and corrosion-promoting dust."
+            }
+          ]
+        },
+        {
+          "description": "t indicates the thickness of the metal that corrodes away annually.",
+          "key": "Corrosion rate",
+          "type": "float",
+          "unit": "(um)(year)^{-1}",
+          "value": null
+        },
+        {
+          "key": "Temperature",
+          "type": "float",
+          "unit": "(Celsius)",
+          "value": null
+        },
+        {
+          "key": "Relative Humidity",
+          "type": "float",
+          "unit": "(%)",
+          "value": null
+        },
+        {
+          "key": "Chloride pollution annual average",
+          "type": "float",
+          "unit": "(mg)(m)^{−2}(days)^{−1}",
```

```suggestion
          "unit": "(mg)(m^{2} days)^{−1}",
```
This was what we agreed upon?

---

Doesn't make a big difference to me. Both look the same. The minus sign brings them to the denominator irrespective of its position.

---

changed.